core-bundle/src/Resources/contao/templates/modules/mod_root_page_dependent_module.html5

```diff
@@ -0,0 +1 @@
+<?= $this->module;
```

Comment:
Do you see any value in having an extra template here? I would just render it directly. We do the same in `ContentModule` for example. There's no point in having just one rendered HTML variable in a template.

Reply:
Fine with rendering directly - didn't think about it. 👍

Reply:
I actually don't mind this. What about adding a specific wrapper to this particular module, e.g. A CSS class that depends on the root page?

Reply:
Fixed in 86cbde127.